src/gauging.jl

```diff
@@ -73,30 +76,22 @@ function vidal_gauge(
   return ψ_vidal, bond_tensors
 end
 
-"""Use an ITensorNetwork ψ in the symmetric gauge and its mts to put ψ into the vidal gauge. Return the bond tensors and ψ_vidal."""
+"""Use an ITensorNetwork ψ in the symmetric gauge and its belief propagation cache to put ψ into the vidal gauge. Return the bond tensors and ψ_vidal."""
 function vidal_gauge(
   ψ::ITensorNetwork,
-  pψψ::PartitionedGraph,
-  mts;
+  bpc::BeliefPropagationCache;
   eigen_message_tensor_cutoff=10 * eps(real(scalartype(ψ))),
   regularization=10 * eps(real(scalartype(ψ))),
   edges=NamedGraphs.edges(ψ),
   svd_kwargs...,
```

Comment:
It seems like you could just define all of these keyword
```suggestion
  kwargs...
```
and then forward all of them to the next call to `vidal_gauge`.